DH_Engine/Classes/DHATGun.uc

```diff
@@ -495,6 +492,7 @@ state Rotating
         if(RotationProjector != none)
         {
             RotationProjector.Destroy();
+            RotationProjector = none;
```

Comment:
This actually isn't necessary. Calling `Destroy` will set the reference to be set to `none` upon next tick, and `bDeleteMe` is set to `true`.